ui/languages

```diff
@@ -2469,8 +2482,8 @@ gab	Gabri
 gad	Gaddang
 gae	Guarequena
 gaf	Gende
-gag	Gagauz
-#gag-Cyrl	Gagauz
+#Gagauz
+gag	Гагаузы
```

Comment:
This entry is incorrect, the reason there is a `-Cyrl` sub-locale in that the default Gagauz alphabet is a Latin based one similar to Turkish.